adminservice/data/goals.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "name": "porn",
+    "categories": [
+      "pornography",
+      "nudity"
+    ]
+  },
+  {
+    "name": "news",
+    "categories": [
+      "news/media",
+      "newsgroups/forums"
+    ]
+  }
+]
```

Comment:
If you put the content here that's being created by the setup test of the Admin service, then our tests continue to work without any changes.